README.md

```diff
@@ -13,6 +13,14 @@ We regression-test the SDK against a selection of Ruby versions (which we update
 
 If you find any compatibility issues, please [do raise an issue](https://github.com/ably/ably-ruby/issues/new) in this repository or [contact Ably customer support](https://support.ably.io/) for advice.
 
+## Known Limitations
+
+This client library is currently *not compatible* with some of the Ably features:
+
+| Feature | Spec reference |
+| :--- | :--- |
+| Custom transportParams | [RTC1f](https://ably.io/documentation/client-lib-development-guide/features/#RTC1f) |
```

Comment:
Please link to the actual docs, and do not use ably.io, our URLs are `www.ably.io`.  Also, one column is fine, see https://github.com/ably/ably-go/commit/ff533763edf288ed3cbe7a7001ba60f76af5f1b0